.github/workflows/update_deps.yml

```
@@ -0,0 +1,26 @@
+name: Monthly dependency updates
+on:
+  schedule:
+    # First of every month
+    - cron: 0 0 1 * *
+
+jobs:
+  create_issue:
+    name: Update dependencies
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+    steps:
+      - name: Update dependencies
+        uses: imjohnbo/issue-bot@v3.3.4
```

GH recommends pinning the actions to their commit sha instead of the tag, since the sha is immutable while the tag can be changed to point to a different commit.

```suggestion
        uses: imjohnbo/issue-bot@7e438653c0da13f4f79678a56bb9ecbd9dcc26ac
```